LayoutTests/inspector/debugger/getFunctionDetails.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html>
+<head>
+<script src="../../http/tests/inspector/resources/inspector-test.js"></script>
+<script>
+function foo() {}
+foo.displayName = "bar";
+
+let boundOnce = foo.bind();
+boundOnce.displayName = "bound";
+
+let boundTwice = boundOnce.bind();
```

Comment:
For completionist purposes, might be good to test binding `this`, arrow functions, and functions with parameters.

Reply:
`Debugger.getFunctionDetails` will only give the source code location and `name`/`displayName`, so there won't be any difference whether we bind `this` or arguments or whatnot

arrow functions is a good idea just in case :)